This patch is based on the commit described below, taken from upstream pull
request 3175, it was adapted for use in the Debian package by Peter Michael
Green.

commit 9fe31f90a01d5c5c49eef81676f5968b917f5e62
Author: Ola x Nilsson <olani@axis.com>
Date:   Mon Mar 20 14:33:33 2023 +0100

    gnu: Update struct aiocb for _TIME_BITS=64

Index: libc/src/unix/linux_like/linux/gnu/mod.rs
===================================================================
--- libc.orig/src/unix/linux_like/linux/gnu/mod.rs
+++ libc/src/unix/linux_like/linux/gnu/mod.rs
@@ -32,7 +32,7 @@ s! {
         __error_code: c_int,
         __return_value: ssize_t,
         pub aio_offset: off_t,
-        #[cfg(all(not(target_arch = "x86_64"), target_pointer_width = "32"))]
+        #[cfg(all(not(gnu_time64_abi), not(target_arch = "x86_64"), target_pointer_width = "32"))]
         __unused1: [c_char; 4],
         __glibc_reserved: [c_char; 32],
     }
